indico/modules/events/registration/models/forms.py

```diff
@@ -173,6 +174,18 @@ class RegistrationForm(db.Model):
         nullable=False,
         default=False
     )
+    #: Whether the registration form is only for selected users
+    private = db.Column(
+        db.Boolean,
+        nullable=False,
+        default=False
+    )
+    uuid = db.Column(
```

Comment:
You need to add the uuid column to this line in the RegistrationFormCloner so it's regenerated when cloning it:

    attrs = get_attrs_to_clone(RegistrationForm, skip={'start_dt', 'end_dt', 'modification_end_dt', 'is_purged'})